server/routes/stats-route.js

```diff
@@ -0,0 +1,14 @@
+// Import express
+const express = require("express");
+
+// Import usage-controller
+const usageRoutes = require("../controllers/usage-controller");
+
+// Create router
+const router = express.Router();
+
+router.get("/systemStats", usageRoutes.getSystemStats);
```

Comment:
`"/systemStats"`-> `"/system"`
We are in the `stats` route, we don't need to specify the obvious.